server/index.js

```diff
@@ -19,44 +19,38 @@ exports.init = function(port, next) {
         routes: { cors: true }
     });
 
+    server.register([Bell], (err) => {
 
-    // Declare an authentication strategy using the bell scheme
-    // with the name of the provider, cookie encryption password,
-    // and the OAuth client credentials.
+        if (err) {
+            throw new Error(err);
+        }
 
-    server.register([Bell, AuthCookie], () => {
-        server.auth.strategy('sparkCookie', 'cookie', {
-            password: 'cookie_encryption_password_secure',
-            cookie: 'user',
-            isSecure: false
-        });
         server.auth.strategy('facebook', 'bell', {
             provider: 'facebook',
             password: 'cookie_encryption_password_secure',
             clientId: '612765462219386',
             clientSecret: '2880756f863270ac97a9500ef80f64a3',
```

Comment:
Fuck, I have just realised, these need to go in our config files, first thing tomorrow lets change the client secret on FB, and then push up the new one. ⛔